cypress/support/e2e.ts

```diff
@@ -5,4 +5,9 @@ declare global {
   }
 }
 
-export {};
+function getPath(location: Location): string {
```

Comment:
Wonder if we could make this based on the env var, so it is more obvious what's going on. In case we decide to keep the env var, that is.

Reply:
0772047e